src/include/sof/debug/telemetry/telemetry.h

```diff
@@ -0,0 +1,86 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
+ */
+
+#ifndef __SOF_TELEMETRY_H__
+#define __SOF_TELEMETRY_H__
+
+/* Slot in memory window 2 to be used as telemetry slot */
+#define DW_TELEMETRY_SLOT 1
+/* Memory of average algorithm of performance queue */
+#define AVG_PERF_MEAS_DEPTH 64
+/* Number of runs taken to calculate average (algorithm resolution) */
+#define AVG_PERF_MEAS_PERIOD 16
+/* disables calculating systick_averages */
+#define PERFORMANCE_MEASUREMENTS
+
+struct system_tick_info {
```

Comment:
external ABI structures ? if so we need packing etc.

Reply:
Other memory window structs have __packed only. I'll add it.